recipes/epics-base/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set version = "3.14.12.8" %}
+{% set sha256 = "b11eab2fdd01c60219be48954631f22230107380961e0df7c356f7cc8bb5f24e" %}
+
+package:
+  name: epics-base
+  version: {{ version }}
+
+source:
+  fn: baseR{{version}}.tar.gz
+  url: https://epics.anl.gov/download/base/baseR{{version}}.tar.gz
+  sha256: {{ sha256 }}
+  patches:
+    - config_site.diff  # [not win]
+    - config_common.diff  # [not win]
+    - rules_build.diff
+    - config_readline.diff  # [linux]
```

Comment:
Please put some explanation regarding these patches

Reply:
@simongregorebner, can you provide some documentation that I can put in?